platform/src/pages/_app.js

```diff
@@ -0,0 +1,5 @@
+import 'common/styles/global.scss';
+
+export default function App({ Component, pageProps }) {
+  return <Component {...pageProps} />;
+}
```

Comment:
Let's use the **.jsx** extension for js files with **JSX** tags.

Reply:
> Thanks @Codebmk I however notice that the deploy preview and test jobs are failing. This is because the yarn.lock file on your PC is outdated. You can fix this by deleting your current yarn.lock file and node_modules, then run `yarn install`
> 
> ![image](https://user-images.githubusercontent.com/41787587/192312306-7e3855a6-a78e-4130-b51c-17dcf65a68bb.png)



Ohh thanks